### ISSUE TITLE
Upgrade to openssl 3.0

### ISSUE THREAD

**Issue by puja-m-patil** [openresty/docker-openresty#236]
Can we upgrade openssl to 3.0 ? How to add it in docker?
when I run command openssl version it shows 3.0 but nginx -V shows openssl 1.1.1u.

**Comment by neomantra** [comment]
No, OpenResty builds against and [bundles its own patched OpenSSL](https://openresty.org/en/openssl-patches.html), so is tightly coupled.  An OpenSSL 3.0 upgrade would come from upstream.   I could't quick-find a related upstream issue in the other OpenResty projects, but I think there is one. 

I'm closing this because there's nothing to do until it exists, and when it does, we will build images for it.
